docs/framework/docker/aspnetmvc.md

```diff
@@ -1,221 +1,8 @@
 ---
-title: Migrating ASP.NET MVC Applications to Windows Containers
-description: Learn how to take an existing ASP.NET MVC application and run it in a Windows Docker Container
-keywords: Windows Containers, Docker, ASP.NET MVC
-author: BillWagner
-ms.author: wiwagn
-ms.date: 02/01/2017
-ms.topic: article
-ms.prod: .net-framework
-ms.technology: dotnet-mvc
-ms.devlang: dotnet
-ms.assetid: c9f1d52c-b4bd-4b5d-b7f9-8f9ceaf778c4
+redirect_url: /aspnet/mvc/overview/deployment/docker-aspnetmvc
```

Comment:
we would want to preserve the comments when we migrate too, so add the following metadata as well:

`redirect_document_id: TRUE`